recipes-devtools/cargo/cargo.inc

```diff
@@ -1,6 +1,6 @@
 INHIBIT_CARGO_DEP = "1"
```

Comment:
Hmm, that's interesting.  You're correct that this doesn't seem to do anything, but I don't understand how we're not ending up with a circular dependency on cargo-native.  It looks like BASEDEPENDS gets dropped out of DEPENDS on a -native build (which probably means if we build a native package using cargo, it wouldn't get the right dependencies).

I'll play with this a little more.  I'm not sure adding to BASEDEPENDS is a kosher thing to do anyway, as that variable isn't actually documented by bitbake...